openmdao/drivers/test/test_scipy_optimizer.py

```diff
@@ -281,7 +281,7 @@ def test_simple_array_comp_SLSQP_scaler_unity_eq(self):
         prob.driver.options['optimizer'] = 'SLSQP'
         prob.driver.add_desvar('x', lower=-50.0, upper=50.0, scaler=np.array([1.0, 1.0]))
 
-        prob.driver.add_objective('o', scaler=np.array([1.0, 1.0]))
+        prob.driver.add_objective('o', scaler=1.0)
```

Comment:
This had nothing to do with the py2/py3 issue but simply came up as a failed test when I ran them. The test was failing saying that the objective has to be a scalar. And unless I've missed something the objective 'o' is a scalar, and not an array with size 2, that was specified as scaler. I can change it back to an array scaler of size 1, but it shouldn't be size 2, right?